terraform/aws/variables.tf

```diff
@@ -104,3 +104,7 @@ variable "vpn_enabled" {
   description = "setup instance for vpn"
 }
 
+variable "main_domain" {
```

Comment:
I wouldn't set the default value since the community will use this tool in order to spin up own devnets.

Reply:
this still can be re-defined via tfvars file.

Reply:
Yes, but we should force people redefine it. And I would do that optional, maybe not everybody need it.

Reply:
this one fixed